pkg/interop/contract/contract.go

```diff
@@ -30,30 +21,6 @@ const (
 	NoneFlag CallFlag = 0
 )
 
-// Create creates a new contract using a set of input parameters:
-//     script      contract's bytecode (limited in length by 1M)
-//     manifest    contract's manifest (limited in length by 2 KiB)
-// It returns this new created Contract when successful (and fails transaction
-// if not). It uses `System.Contract.Create` syscall.
-func Create(script []byte, manifest []byte) *Contract {
```

Comment:
It is not a critical issue, but `Create`, `Update` and `Destroy` may be removed from the list of available compiler's syscalls, see `pkg/compiler/syscall.go`. As long as from `interopnames`, we don't have such interops now.

Reply:
Fixed